array_contains throws exception if array in colelction is none

..\..\venv\Lib\site-packages\parameters_validation\validate_parameters_decorator.py:37: in wrapper
    return f(*args, **kwargs)
..\..\svc\db\db.py:94: in get_tails_for_user
    docs = tails_ref.stream()
..\..\venv\Lib\site-packages\mockfirestore\query.py:31: in stream
    doc_snapshots = [doc_snapshot for doc_snapshot in doc_snapshots
..\..\venv\Lib\site-packages\mockfirestore\query.py:32: in <listcomp>
    if compare(doc_snapshot._get_by_field_path(field), value)]
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

x = None, y = 'red'

>   return lambda x, y: y in x
E   TypeError: argument of type 'NoneType' is not iterable

..\..\venv\Lib\site-packages\mockfirestore\query.py:137: TypeError

--- a/mockfirestore/query.py
+++ b/mockfirestore/query.py
@@ -145,7 +145,7 @@ class Query:
         elif op == 'in':
             return lambda x, y: x in y
         elif op == 'array_contains':
-            return lambda x, y: y in x
+            return lambda x, y: x is not None and y in x
         elif op == 'array_contains_any':
             return lambda x, y: any([val in y for val in x])
 

